serverless.yaml

```diff
@@ -174,11 +174,21 @@ functions:
     memorySize: 192
     runtime: nodejs12.x
     description: 'Run the crawler for bulk export'
-    role: StartCrawlerLambdaRole
+    role: CrawlerRelatedLambdaRole

```

Comment:
Should we be sharing the role across lambdas? seems like we are giving more permissions than required -- unsure though

Reply:
Ideally no. I split it into 2 roles.